### DISCUSSION TITLE
Default delete option

### DISCUSSION THREAD

**Discussion by pablo03v** [ente-io/ente#848] [Enhancements]
**Describe the feature**
Instead of asking me if I want to delete the photo from both, ente, or the device I'd love to set a default option :D

![signal-2023-11-24-06-07-57-247](https://github.com/ente-io/photos-app/assets/93644977/9ad954a0-87d9-497d-b2b9-301f7328f649)

**Comment by noaho** [comment]
Same, I'm trying to set this up for other people and I know if they have to make a decision everytime they delete something it's going to lead to files deleted from either library and a huge mess..

I like that if I set ente with photo and video access I don't get prompted twice after the initial prompt. Someone suggested that in the discord.

**Comment by kaidelorenzo** [comment]
I agree with this however I think if a user sets the delete option default to both it would be important to have a really convenient "undo" delete button in case of accidental taps.

**Reply by pablo03v** [reply]
Yes, a trash which empties itself every x (30?) days
